app/models/user.rb

```diff
@@ -22,4 +22,13 @@ def self.enabled_merchants
   def self.default_users
     where(role: 0)
   end
+
+  def toggle_status(user)
```

Comment:
This method doesn't need to take a user as an argument since you can just call it on the instance itself, replacing `user` with `self` inside the method should work.